wca-regulations.md

```diff
@@ -252,8 +252,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 9f13) For "Mean of 3" and "Average of 5" rounds, rankings are assessed based on the ordering of the averages/means of the competitors, where "better" is the shorter recorded time.
     - 9f14) For "Mean of 3" and "Average of 5" rounds, if two or more competitors achieve identical average/mean results, rankings are assessed based on the best attempt per competitor, where "better" is defined as the shorter time.
     - 9f15) Competitors who achieve the same result in a round receive an identical ranking for the round.
-- 9g) A Combined Round consists of two phases of attempts, where competitors advance to the second phase if they meet a designated cutoff during the first phase.
-    - 9g2) Whether a competitor proceeds to next phase of a Combined round must be decided by result (all competitors with a better result than x) of the first phase (see [Regulation 9f12](regulations:regulation:9f12)).
+- 9g) If the round format is "Average of 5", a round may have a "Best of 2" cutoff phase where the competitor must meet a given cutoff for either of the first two solves (e.g. a result better than X seconds) in order to be eligible for the remaining three attempts.
+    - 9g1) Any cutoff must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
```

Comment:
@thewca/wrc-team I believe that cutoffs are sometimes changed in practice, so I'm copying the regulation for time limits. Let me know if you think we need to lock down cutoffs (but not time limits) more than trusting the judge's discretion.